usaspending_api/awards/models.py

```diff
@@ -310,6 +328,24 @@ def get_default_fields(path=None):
             "assistance_data"  # must match related_name in TransactionAssistance
         ]
 
+    @classmethod
+    def get_or_create(cls, **kwargs):
```

Comment:
I don't have overly-strong feelings. We already have a `get_or_create_summary_award` method on `Award`, so precedent for doing that, even though I appreciate your initial approach to being less verbose.

Given that there's an argument for either name, might was well keep what you have.